docs/en/modules/tmr.md

```diff
@@ -235,25 +244,29 @@ If the specified timer is registered, returns whether it is currently started an
 
 #### Example
 ```lua
-running, mode = tmr.state(0)
+mytimer = tmr.create()
+print(mytimer:state()) -- nil
+mytimer:register(5000, tmr.ALARM_SINGLE, function() print("hey there") end)
+running, mode = mytimer:state()
+print("running: " .. tostring(running) .. ", mode: " .. mode) -- running: false, mode: 0
 ```
 
 ## tmr.stop()
 
 Stops a running timer, but does *not* unregister it. A stopped timer can be restarted with [`tmr.start()`](#tmrstart).
 
 #### Syntax
-`tmr.stop(id/ref)`
+`tmr.stop([id/ref])`
 
 #### Parameters
-`id`/`ref` timer id (0-6) or object
+`id`/`ref` timer id (0-6) or object, obsolete for OO API (→ [`tmr.create()`](#tmrcreate))
 
 #### Returns
 `true` if the timer was stopped, `false` on error
 
 #### Example
 ```lua
-if not tmr.stop(2) then print("timer 2 not stopped, not registered?") end
+if not tmr.create():stop() then print("timer not stopped, not registered?") end
```

Comment:
Not really, you're right. But the existing example (by itself) was just as meaningless. It's an example to demonstrate "`false` on error"...How would you like to see this improved?